src/app/core/view/fa-dynamic-icon/fa-dynamic-icon.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, Input } from "@angular/core";
+import { IconDefinition, IconName } from "@fortawesome/fontawesome-svg-core";
+import {
+  faChartLine,
+  faChild,
+  faHome,
+  faQuestionCircle,
+  faTable,
+  faUniversity,
+  faWrench,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  faCalendar,
+  faCalendarCheck,
+  faFileAlt,
+  faUser,
+} from "@fortawesome/free-regular-svg-icons";
+import { FaIconLibrary } from "@fortawesome/angular-fontawesome";
+
+/**
+ * font-awesome icons decoupled from the config
+ */
+const faDynamicIcons = new Map<string, IconDefinition>([
+  ["home", faHome],
+  ["child", faChild],
+  ["university", faUniversity],
+  ["calendar", faCalendar],
+  ["calendar-check", faCalendarCheck],
+  ["table", faTable],
+  ["notes", faFileAlt],
+  ["wrench", faWrench],
+  ["user", faUser],
+  ["question", faQuestionCircle],
+  ["line-chart", faChartLine],
+]);
+
+/**
+ * This component can be used to display dynamic Font-Awesome icons.
+ * The term 'dynamic icons' refers to icons that are injected at runtime,
+ * for example through the config.
+ */
+@Component({
+  selector: "app-fa-dynamic-icon",
+  template: `<fa-icon [icon]="_icon"></fa-icon>`,
+})
+export class FaDynamicIconComponent {
+  /** The fallback icon if the given icon is neither known (inside the internal map)
+   * nor registered as a font-awesome icon
+   */
+  static fallbackIcon = faQuestionCircle;
+
+  /**
+   * Sets the dynamic icon by name.
+   * You should make sure that the icon is registered inside the {@link faDynamicIcons}-map,
+   * or put it into this map if it isn't there.
+   * <br>
+   * If for some reason the icon is not inside the map or cannot be inserted into the map,
+   * the icon name has to match the <em>exact</em> icon name as provided by font-awesome.
+   * Prefixed icon names are not supported and the prefix is always assumed to be `far`
```

Comment:
I guess `far` refers to the font awesome regular library. Which icons belong to the font awesome solid map and are therefore no that easily accessible?

Reply:
Most of these icons are regular icons that have a "filled" look and thus exist both with `far` and `fas`. I had a quick look at how many icons exist in both definitions and I think that it would actually be better to use the solid icons since there are way more of them

Reply:
we should be consistent, although I am not sure that the number of choices is the only important aspect. Let's briefly rope in Rosa and compare the look and feel of solid vs. outlined icons.